cob_hardware_config/common/upload_robot.launch

```diff
@@ -1,13 +1,10 @@
 <?xml version="1.0"?>
 <launch>
 
-    <arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
```

Comment:
Tabs are really annoying..
(my default editor does not handle them properly :-/)